hs-backend-booster/pom.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!-- Copyright (c) Runtime Verification, Inc. All Rights Reserved. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.runtimeverification.k</groupId>
+    <artifactId>parent</artifactId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>hs-backend-booster</artifactId>
+  <packaging>jar</packaging>
+
+  <name>K Framework Haskell Backend Booster</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.runtimeverification.k</groupId>
+      <artifactId>kernel</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+  </dependencies>
+
+  <properties>
+    <stack.extra-opts value="" />
+  </properties>
+
+  <profiles>
+    <profile>
+      <id>haskell.backend.booster.Release</id>
+      <activation>
+        <property>
+          <name>project.build.type</name>
+          <value>Release</value>
+        </property>
+      </activation>
+    </profile>
+    <profile>
+      <id>haskell.backend.booster.RelWithDebug</id>
+      <activation>
+        <property>
+          <name>project.build.type</name>
+          <value>RelWithDebug</value>
+        </property>
+      </activation>
+    </profile>
+  </profiles>
+
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <version>1.7</version>
+        <executions>
+          <execution>
+            <id>build-haskell-booster</id>
+            <phase>compile</phase>
+            <configuration>
+              <skip>${haskell.backend.booster.skip}</skip>
```

Comment:
Can we make this shorter?
Something like `booster.skip`?
It's going to make it harder to type. And I tend to do that quite often when I need to test different versions.

Reply:
Sure!